eth/tracers/api.go

```diff
@@ -324,15 +315,37 @@ func (api *API) traceChain(ctx context.Context, start, end *types.Block, config
 				log.Info("Tracing chain segment", "start", start.NumberU64(), "end", end.NumberU64(), "current", number, "transactions", traced, "elapsed", time.Since(begin))
 			}
 			// Retrieve the next block to trace
```

Comment:
// Retrieve the next block to trace -> // Retrieve the parent state to trance on top